tracer/build/_build/Build.cs

```diff
@@ -320,7 +320,7 @@ void DeleteReparsePoints(string path)
                 (rid: "linux-x64", archiveFormat: ".tar.gz"),  
                 (rid: "linux-musl-x64", archiveFormat: ".tar.gz"),  
                 (rid: "osx-x64", archiveFormat: ".tar.gz"),  
```

Comment:
@tonyredondo did you mean to add an osx-arm64 build in this PR?

Reply:
Yes but I had to remove it. Building the standalone tool is quite complicated. We are currently building using netcoreapp3.1, and osx-arm64 support starts from net6.0. If we switch to net6.0 for then we start having problems with the trimmer. So because we don’t have a way to build for M1 in CI, I decided to revert that change and pospone it until we solve the trimming issue.